docker/docker_nightly.py

```diff
@@ -55,3 +60,11 @@
     # Push images with latest tag
     try_and_handle(f"docker push {organization}/{project}:latest-cpu", dry_run)
     try_and_handle(f"docker push {organization}/{project}:latest-gpu", dry_run)
+
+    # Cleanup built images
+    if args.cleanup:
```

Comment:
is this needed? 

I think something like `docker system prune --all --volumes` should work instead, otherwise now people might miss a few chars here and there

There are some well known docker nuke commands online, maybe we should bring something in

Reply:
Tried it. It works! Updated the code.